test/Sema/class_differentiable.swift

```diff
@@ -0,0 +1,570 @@
+// SWIFT_ENABLE_TENSORFLOW
```

Comment:
These tests are not the best, I directly copied `test/Sema/struct_differentiable.swift`.
Todo: clean up and tighten tests. Fuzzing would be cool.